pydicom/tests/test_dataset.py

```diff
@@ -1896,3 +1920,72 @@ def test_copy(self, copy_method):
         assert ds_copy.is_implicit_VR
         assert ds_copy.is_little_endian
         assert ds_copy.read_encoding == "utf-8"
+
+
+CAMEL_CASE = (
+    [  # Shouldn't warn
+        "Rows", "_Rows", "Rows_", "rows", "_rows", "__rows", "rows_", "ro_ws",
+        "rowds", "BitsStored", "bits_Stored", "Bits_Stored", "bits_stored",
+        "_BitsStored", "BitsStored_", "B_itsStored", "BitsS_tored",
+        "12LeadECG", "file_meta", "filename", "is_implicit_VR",
```

Comment:
Hmm, weird. Why doesn't 12LeadECG raise SyntaxError?

Reply:
Interesting...

Reply:
Hmm, yeah, confirmed in normal Python class - can only get back out with `getattr()`